### PR TITLE
Add coverage support

### PR DESCRIPTION
**Pull request by vaibhavmule**
We already have coverage folder ignored in .gitignore

A coverage report is shown in terminal and written to `./coverage` when you run `npm run coverage`

image

### CODE REVIEW

**Review by vaibhavmule — COMMENTED** [review]
@Albert-Gao 

We have two options here.

* Just keep it as `npm run coverage`, Also, we can rename `coverage` to `cover` for better. 
* We can use configuration and can add coverage by default when you run `npm run test` , please see https://facebook.github.io/jest/docs/en/configuration.html#collectcoverage-boolean

**Comment by Albert-Gao** [comment]
If we go to the separate command, I'd like to use `coverage`, 'cos I think it's clear.

And I think it might be better if we just update the `doc` to let people know that they can use `npm run test -- --coverage`, because if we go to `npm run coverage`, it's really testing and collecting coverage, a little bit duplicate to me.

What do you think? :)

But still It's a little bit strange that it failed the check. I tried to re-run, but still the same. Maybe encoding?

**Comment by vaibhavmule** [comment]
Yeah I had same feeling, the duplication part.
Lets go with this `npm run test -- --coverage`.

Travis, yesterday there was an issue with Travis I saw on their status page, anyways

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Just added 1 more:
 - `npm run clean` : Clean the `./dist` folder
 - `npm run build`: Clean and build `src` to `dist`
 - `npm run test`: Run tests via `Jest`
-- `npm run coverage`: Generates test coverage report via `Jest`
+- `npm run test -- --coverage`: Generates test coverage report via `Jest`
```
Comment (vaibhavmule) [inline README.md]: @Albert-Gao 

Feel free to edit the description of the command.

**Pull request merged**